flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -227,6 +227,14 @@
 	public static final String TASK_MANAGER_NETWORK_NUM_BUFFERS_KEY = "taskmanager.network.numberOfBuffers";
 
 	/**
+	 * Boolean flag to enable/disable more detailed metrics about inbound/outbound network queue lengths
+	 */
+	@PublicEvolving
+	public static final ConfigOption<Boolean> NETWORK_DETAILED_METRICS_KEY =
```

Comment:
I don't think ConfigOptions are supposed to be in this class; itshould be moved to either ```NetworkOptions``` or ```MetricOptions```.